packages/ui/src/views/pull-request/pull-request-list.tsx

```diff
@@ -30,61 +30,65 @@ export function PullRequestList({ pullRequests, open_prs, closed_prs }: PullRequ
     })
   }, [headerFilter])
 
+  const onOpenClick = () => {
+    setHeaderFilter('open')
+  }
+  const onCloseClick = () => {
+    setHeaderFilter('closed')
+  }
+  if (!filteredData?.length) return null
```

Comment:
Some line breaks would make this easier to visually parse at a glance. Also since you know `filteredData` exists and will be an array, you don't need the optional chaining.
```suggestion
  }

  if (!filteredData.length) return null

```